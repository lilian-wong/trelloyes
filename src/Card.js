import React from 'react';
import './Card.css';

class Card extends React.Component{
    constructor(props) {
        super(props);
        this.state = {
            title: props.title,
            content: props.content
        };
    }

    render(){
        return (
            <div className="Card">
                <button type="button">delete</button>
                <h3>{this.props.title}</h3>
                <p>{this.props.content}</p>
            </div>
        )
    }
}


export default Card