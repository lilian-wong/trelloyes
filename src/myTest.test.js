import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';


it('render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    //ReactDOM.unmountComponentAtNode('div');
});

it('working', ()=>{
    const tree = renderer
        .create(<div>Here</div>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});