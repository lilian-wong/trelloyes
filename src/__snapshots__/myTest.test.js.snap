// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`working 1`] = `
<div>
  Here
</div>
`;
